Sources/MapboxNavigation/CarPlayManagerDelegate.swift

```diff
@@ -147,6 +148,19 @@ public protocol CarPlayManagerDelegate: AnyObject, UnimplementedLogging {
      - parameter navigationViewController: The CarPlayNavigationViewController that was presented on the CarPlay display.
      */
     func carPlayManager(_ carPlayManager: CarPlayManager, didPresent navigationViewController: CarPlayNavigationViewController) -> ()
+    
+    /**
+     Tells the receiver that the final destination `PointAnnotation` was added to either `CarPlayMapViewController` or `CarPlayNavigationViewController`.
+     
+     - parameter carPlayManager: The `CarPlayManager` object.
+     - parameter finalDestinationAnnotation: `PointAnnotation`, which was added to the `MapView`.
+     - parameter parentViewController: `UIViewController`, which contains `MapView` instance. Can be either `CarPlayMapViewController` or `CarPlayNavigationViewController`.
+     - parameter pointAnnotationManager: `PointAnnotationManager` instance, which is responsible for `PointAnnotation`s management in the `NavigationMapView`.
+     */
```

Comment:
Nit: Sometimes it’s easier to read a method’s documentation if it uses fewer formal symbol references to types. These type references are also prone to getting out of sync during refactors, such as `NavigationMapView` versus `MapView`. The developer can still look at the method declaration to see what the types should be. That said, cross-references to related methods and types that aren’t obvious from the declaration are very much a good thing.

```suggestion
    /**
     Tells the receiver that the `PointAnnotation` representing the final destination was added to either `CarPlayMapViewController` or `CarPlayNavigationViewController`.
     
     - parameter carPlayManager: The `CarPlayManager` object.
     - parameter finalDestinationAnnotation: The point annotation that was added to the map view.
     - parameter parentViewController: The view controller that contains the map view, which is an instance of either `CarPlayMapViewController` or `CarPlayNavigationViewController`.
     - parameter pointAnnotationManager: The object that manages the point annotation in the map view.
     */
```

Reply:
Yep, I guess using more generic naming can save some time in future when symbols are renamed.